pkgs/development/compilers/llvm/8/libunwind.nix

```
@@ -1,4 +1,4 @@
-{ stdenv, version, fetch, cmake, fetchpatch }:
+{ stdenv, version, fetch, cmake, fetchpatch, enableShared ? true }:
```

The modified `stdenv` for `pkgsStatic` was recently extended to carry a `static` flag, this can be used for the default argument, making the explicit override in the static overlay unnecessary.
```suggestion
{ stdenv, version, fetch, cmake, fetchpatch, enableShared ? !(lib.hasAttr "static" stdenv) }:
```
You could even take it one step further and remove `enableShared` and check the implicit parameter in the cmakeFlags directly.

---

We definitely want these to be overridable, so providing enableShared is important.

---

Do we? What is the use case for specifying this at individual package granularity?

---

Perhaps it's not so important. Some users might want to fine tune which packages are static and which are shared (in some cases like kerberos, no static version is available). It's nice to have a specific flag for this, although you can always just override the stdenv being used.